### PR TITLE
remove matrix

### PR DESCRIPTION
**Pull request by MattesR**
and fix static path

### CODE REVIEW

File: liquidcore/site/settings.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def bool_env(value):
 USE_L10N = True
 USE_TZ = True
 STATIC_URL = '/static/'
-STATIC_ROOT = str(base_dir / 'static')
+STATIC_ROOT = str(base_dir / 'liquidcore' / 'home' / 'static')
```
Comment (mgax) [inline liquidcore/site/settings.py:110]: I'm not sure this is correct. [`STATIC_ROOT`](https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-STATIC_ROOT) is meant for deployment, and Django expects to copy the static files over there as part of the deployment steps. So it shouldn't be pointing back to the source code.

**Review by mgax — CHANGES_REQUESTED** [review]
Hey, yes, this is a great idea! I've been removing some of these things over in my other PR, https://github.com/liquidinvestigations/core/pull/44, but really they should be a separate thing.

I think you should cherry-pick matrix-related and static-related changes from my PR into yours and I'll rebase after we merge yours.

File: liquidcore/site/settings.py (hunk 1 above)
Comment (MattesR) [reply liquidcore/site/settings.py:110]: I've shelled into the container to find out where the static files are actually sitting and it's in that folder. The Dockerfile copies the folder into `./liquidcore`
`ADD liquidcore ./liquidcore`
Edit: I've deployed the latest version of the core and everything seems to be in the right place, so I'll just cloes this PR.

Comment (mgax) [reply liquidcore/site/settings.py:110]: Those are files from one django app. There could, in principle, be more. The `collectstatic` command collects static files from all apps and copies them into a single folder so they can be easily served by e.g. nginx.

**Pull request closed without merging**